TF_Settings_Web/src/Pages/Visuals/ColorPicker.tsx

```diff
@@ -37,19 +36,12 @@ const ColorPicker: React.FC<ColorPickerProps> = ({ cursorStyle, updateCursorStyl
                 <HexAlphaColorPicker
                     color={cursorStyle[activeTabIndex]}
                     onChange={(color) => updateCursorStyle({ ...cursorStyle, [activeTabIndex]: color })}
-                    onPointerUp={() => writeOutConfig()}
                 />
                 <input
                     type="text"
                     className={'color-picker__body__text'}
                     value={cursorStyle[activeTabIndex].toUpperCase()}
                     onChange={(e) => updateCursorStyle({ ...cursorStyle, [activeTabIndex]: e.target.value })}
-                    onBlur={() => writeOutConfig()}
```

Comment:
👀 It's now possible to edit the custom color hex string then choose a different tab to leave the page without writing that update. 

Solution could be to always write out the config on navigating away from the page (DM if unsure about this)

Reply:
Changed to listen to `onpointerup` for the whole window now - should resolve this?